aliasmgr/aliasmgr.go

```diff
@@ -390,7 +412,7 @@ func (m *Manager) DeleteLocalAlias(alias,
 	// cache.
 	delete(m.aliasToBase, alias)
 
-	return nil
+	return m.LinkUpdater(baseScid)
```

Comment:
We still hold the lock here, which will cause a deadlock. See my diff above for a fix.